_docs/prerequisites/mountpropagation.md

```diff
@@ -8,17 +8,39 @@ module: prerequisites/mountpropagation
 # Mount propagation
 
 StorageOS requires mount propagation enabled to present devices as volumes for
-containers. According to the docker installation, that functionality may or may
-not be enabled by default.
+containers (see linux kernel documentation
+[here](http://man7.org/linux/man-pages/man2/mount.2.html)).
 
-On the other hand, orchestrators such as Kubernetes or OpenShift have their own
-implementations. K8S 1.10 and OpenShift 3.10 have mount propagation enabled by
+Certain versions of docker ship with a systemd manifest with
+[MountFlags](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#) set
+to 'slave', thus preventing StorageOS from working. This can be removed or set
+to 'shared' with a simple systemd drop in:
+
+```bash
+mkdir -p /etc/systemd/system/docker.service.d/
+cat <<EOF > /etc/systemd/system/docker.service.d/mount_propagation_flags.conf
+[Service]
+MountFlags=shared
+EOF
+
+# systemctl daemon-reload
+# systemctl restart docker.service
+```
+
+To confirm behaviour, the following command should run without error
+
+```
+docker run -it --rm -v /mnt:/mnt:shared busybox sh -c /bin/date
+```
+
+Orchestrators such as kubernetes or openshift have their own ways of exposing
```

Comment:
Capitalise "Kubernetes or Openshift"

Reply:
Actually, Openshift should be "OpenShift"

Reply:
Pushed a commit which makes these consistent across the site.